server/views/pages/scan/manual-barcode-entry.njk

```diff
@@ -55,14 +55,14 @@
           </div>
         </form>
 
-        <h3 class="govuk-heading-m">What to do if you cannot enter the barcode number</h3>
+        <h2 class="govuk-heading-m">What to do if you cannot enter the barcode number</h2>
         <p>If you haven't been able to enter the barcode number, including if it's damaged or hasn't printed properly,
         follow the link.</p>
         <a href="manually-enter-barcode/report-problem" class="govuk-link" id="report-problem">I cannot enter the barcode number</a>
       </div>
 
       <div class="govuk-grid-column-one-third example-barcode">
-        <h3 class="govuk-heading-m" style="border-top: 10px solid #1d70b8;">Where to find the barcode number</h3>
+        <h2 class="govuk-heading-m" style="border-top: 10px solid #1d70b8;">Where to find the barcode number</h2>
```

Comment:
Can we move this styling into a css file at some point please?

Reply:
Definitely - I hadn't realised I'd left it there TBH!
Next time we are touching this template we should do it 👍